wpilibj/src/athena/java/edu/wpi/first/wpilibj/IterativeRobot.java

```diff
@@ -307,5 +307,6 @@ public void testPeriodic() {
       System.out.println("Default IterativeRobot.testPeriodic() method... Overload me!");
       m_tmpFirstRun = false;
     }
+    Timer.delay(0.001);
```

Comment:
You're right, we made this change late in build season while people weren't thinking clearly.